chapters/annotations.tex

```diff
@@ -1744,6 +1744,8 @@ \subsection{Connector Sizing}\label{connector-sizing}\annotationindex{connectorS
 
 If \lstinline!connectorSizing = true!, a tool may set the variable value in a modifier automatically, if used as dimension size of a vector of connectors.
 In that case the value modifier should not be edited directly by the user, and a tool may choose to not display that variable in the dialog or display it with disabled input field.
+There should be one vector of connectors using the dimension size \lstinline!n! for each variable \lstinline!n! with \lstinline!connectorSizing = true!.
+If there are other vector of connectors that should have the same size, but not cause an automatic update they can use \lstinline!n+0! instead.
```

Comment:
Avoid use of _should_ here:

```suggestion
Other vectors of connectors that need to have the same size may use a different expression with the same value, such as \lstinline!n + 0!, to avoid that multiple vectors become associated with the \lstinline!connectorSizing! automatic updating of \lstinline!n!.
```